string radar_ip
string local_ip
string gateway

uint16      pc_receive_port     # pc断接受端口
uint16      radar_receive_port  # 雷达接受端口
uint16      radar_send_port     # 雷达发送端口

